hack/collector-harvesteros

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -ux
+
+HOST_PATH=$1
+BUNDLE_DIR=$2
+
+cd ${BUNDLE_DIR}
+# get some host information
+cp ${HOST_PATH}/etc/hostname .
+
+###############################################################################
+# collect configs
+###############################################################################
+mkdir -p configs
+cp -r /host/oem configs
+
+###############################################################################
+# collect logs
+###############################################################################
+mkdir -p logs
+cd logs
+
+chroot $HOST_PATH /usr/bin/journalctl -k > kernel.log
+
+units=(rke2-server rke2-agent rancherd rancher-system-agent wicked iscsid)
```

Comment:
Note: in the future we need to detect OS and Kubernetes distro separately, but that can be done later.